modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/action/EditCPDefinitionMVCActionCommand.java

```diff
@@ -162,6 +162,8 @@ else if (cmd.equals("updateConfiguration")) {
 			else if (cmd.equals("updateSubscriptionInfo")) {
 				updateSubscriptionInfo(actionRequest, cpDefinition);
 
+				cpDefinition = _updateCPDefinition(actionRequest, cpDefinition);
```

Comment:
this I believe should be changed into a callable and the _updateCPDefinition call should be moved into that new callable

Reply:
@andrea-ale-sbarra ,

Trying to confirm if this is what you are expecting with the existing callable:

```
			else if (cmd.equals("updateSubscriptionInfo")) {
				updateSubscriptionInfo(actionRequest, cpDefinition);

				Callable<Object> cpDefinitionConfigurationCallable =
					new CPDefinitionConfigurationCallable(
						actionRequest, cpDefinition);

				TransactionInvokerUtil.invoke(
					_transactionConfig, cpDefinitionConfigurationCallable);
```